src/main.rs

```diff
@@ -251,3 +257,14 @@ fn get_leading_zero_bits(bytes: &[u8]) -> u8 {
     }
     res
 }
+
+fn print_qr(secret_key: SecretKey) {
+    let private_hex = secret_key.display_secret().to_string();
+    let code = QrCode::new(private_hex).unwrap();
```

Comment:
Can you please show the `nsec` instead hex private key? we are moving to work with `npub/nsec`.